File: Retos/Reto #12 - VIERNES 13 [Fácil]/python/mouredev.py
import datetime


def friday_13(year: int, month: int) -> bool:
    try:
        return datetime.date(year, month, 13).weekday() == 4
    except:
        return False


print(friday_13(2023, 3))
print(friday_13(2023, 1))
print(friday_13(2023, 13))
print(friday_13(-2023, 1))
print(friday_13(2023, "1"))
print(friday_13(2023, 0))
print(friday_13("Brais", "Moure"))

File: Retos/Reto #12 - VIERNES 13 [Fácil]/javascript/mouredev.js
const friday13 = (year, month) => {
    return new Date(year, month - 1, 13).getDay() === 5;
}

console.log(friday13(2023, 3));
console.log(friday13(2023, 1));
console.log(friday13(2023, 13));
console.log(friday13(-2023, 1));
console.log(friday13(2023, "1"));
console.log(friday13(2023, 0));
console.log(friday13("Brais", "Moure"));